cumulus/parachains/runtimes/collectives/collectives-polkadot/src/ambassador/tracks.rs

```diff
@@ -0,0 +1,286 @@
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! The Ambassador Program's referenda voting tracks.
+
+use super::Origin;
+use crate::{Balance, BlockNumber, RuntimeOrigin, DAYS, DOLLARS, HOURS};
+use pallet_referenda::{StaticTracksIter, Track};
```

Comment:
I dont find the definition of `StaticTracksIter` anywhere?!

Reply:
Leftover of a previous version, will ✂️